contributing/release-and-publish.md

```diff
@@ -0,0 +1,29 @@
+[&larr; Table of Contents](../CONTRIBUTING.md)
+
+# Releases and Publishing
+🛠 How the build and publish pipeline works
+
+## Managing [semver](https://semver.org/) in FDS
+When making changes, bump the semantic version in your branch `Jenkinsfile`.
+Full instructions for doing this can be found [here](https://github.com/cbinsights/form-design-system/#versioningpublishing).
+
+
+## Build pipeline
+On every push to `master`, a CI tool will perform these actions:
+
+1. full build
+2. lint
+3. test
+4. bump the version in `package.json`
+5. publish to npm
+6. tag the release and push the git tag to `origin`
+
+
+## Publishing docs
+All FDS documentation is published to [Github Pages](https://pages.github.com/).
+
+The publishing source is the `docs/` directory in repo root. Documentation is regenerated
+on every full build and **we commit generated documentation**.
+
+Generated documentation is suppressed in pull requests diffs by the `linguist-generated`
+gitattribute.
```

Comment:
hmm... never heard of that before!

We can potentially add more explanation here: "We generate documentation locally which we commit. We generally don't want to see this documentation in our pull request diffs as it complicates things. Our documentation uses a `linguist-generated` gitattribute which suppresses the files being showed in the diff: https://help.github.com/en/articles/customizing-how-changed-files-appear-on-github

We do this because we commit the generated files for now, which means we don't want to use gitignore" and then add a link that explains more about that attribute: https://help.github.com/en/articles/customizing-how-changed-files-appear-on-github

Reply:
Added some clarity in this section